### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by tyhan**
개발 편의를 위한 dockerfile 추가

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+RUN pip3 install --upgrade pip
+
+#RUN pip3 install --no-cache terminal-translator
+
+#for Dev
+ADD . /translate-in-terminal
+WORKDIR /translate-in-terminal
+RUN pip3 install wheel
+RUN python3 setup.py bdist_wheel
+RUN pip3 install dist/terminal_translator-1.0.1-py3-none-any.whl -I
```
Comment (jybaek) [inline Dockerfile]: Can not you change the _version (1.0.1)_ part to a variable? `translate/__init__.py`.

File: translate/translate.py [hunk 2]
```diff
@@ -33,11 +34,11 @@ def main():
     if lang not in ('ko', 'en'):
         raise ValueError('Unknown language')
     data = translator.translate(text, dest=LANG[lang]).text
-    pyperclip.copy(data)
+    if args.clipboard:
```
Comment (jybaek) [inline translate/translate.py]: This removes the ability for the translation to be automatically saved to the _clipboard_. The `args.clipboard` option has a different meaning.

File: Dockerfile (hunk 1 above)
Comment (tyhan) [reply Dockerfile]: Change build steps.

d532d995b87f8833cbe43841488a5515194c74ac

File: translate/translate.py (hunk 2 above)
Comment (tyhan) [reply translate/translate.py]: this part is rolled back.
fed402e5504c408c1f83c0fff174f07fdbb3d98b

**Comment by jybaek** [comment]
thanks!

**Pull request merged**